lib/deflation.cpp

```diff
@@ -0,0 +1,527 @@
+#include <deflation.h>
+#include <qio_field.h>
+#include <string.h>
+
+#ifdef DEFLATEDSOLVER
+
+#ifdef MAGMA_LIB
+#include <blas_magma.h>
+#endif
+
+#include <Eigen/Dense>
+#include <Eigen/Sparse>
+
+#endif
+
+//#define USE_BLOCK_DEFLATION
```

Comment:
this is an artifact, there was a "serial" version. Deflation methods supposed to be "blocked" now using multi-src dslash etc. . Will remove.